guide/14-deep-learning/mistral_workflow_with_text_models.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>Specify the minimum number of examples to 6</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
done